DefaultSettings.php

```diff
@@ -1,26 +1,17 @@
 <?php
 
-namespace SMW;
-
 /**
- * SemanticMediaWiki default settings
+ * DO NOT EDIT!
```

Comment:
The rename of this file needs to be reflected in the docs. I'm thinking this is better done after the next release, rather than right before when an RC already happened.